pinax/stripe/actions/sources.py

```diff
@@ -57,7 +57,7 @@ def sync_card(customer, source):
         address_zip=source["address_zip"] or "",
         address_zip_check=source["address_zip_check"] or "",
         brand=source["brand"],
-        country=source["country"] or "",
+        country=source["country"],
```

Comment:
@paltman as you asked me I had added test to validate None country value. check 7eac4a887a8cbdd2f555f6eb05e3588df7258fe2 for it.

also I had to redo migration in order to avoid conflict with other migration after merge with master (35416e8862a073337f06efe72a53d38a9100e8f0).